activerecord/CHANGELOG.md

```diff
@@ -1,3 +1,21 @@
+*   Add `ActiveRecord::Base.destroy_later` for automatically destroying records
+    after a specified amount of time.
+
+    ```ruby
+    class Export < ApplicationRecord
+      # Destroy all exports 30 days after creation
+      destroy_later after: 30.days
+
+      # Destroy exports 30 days after completing them
+      destroy_later after: 30.days, if: -> { completed_previously_changed? && completed? }
+
+      # Destroy exports 30 days after completing them, ensuring they're still completed at destroy time
+      destroy_later after: 30.days, if: -> { completed_previously_changed? && completed? }, ensuring: :completed?
```

Comment:
Just ported this to an app I'm working on and noticed that `completed_previously_changed?` should be `status_previously_changed?` (or whatever the underlying column is called). 😊